packages/contracts/tasks/helpers/types.ts

```diff
@@ -718,6 +718,11 @@ export interface IVerificationSubtaskArgs {
    * Libraries
    */
   libraries?: Libraries;
+
+  /**
+   * Whether force verification or not
+   */
+  force?: boolean;
```

Comment:
What is this change for?

Reply:
It's just a force flag for verification. This field was missing for this interface.